src/js/compat.ts

```diff
@@ -234,3 +234,22 @@ async function nodeLoadScript(url: string) {
     await import(/* webpackIgnore: true */ nodeUrlMod.pathToFileURL(url).href);
   }
 }
+
+function nodeBase16ToBase64(b16: string): string {
+  return Buffer.from(b16, "hex").toString("base64");
+}
+
+function browserBase16ToBase64(b16: string): string {
```

Comment:
Let's officially drop node 14 support after the release?

Reply:
It does say on the node `atob` api docs:

> This function is only provided for compatibility with legacy web platform APIs and should never be used in new code, because they use strings to represent binary data and predate the introduction of typed arrays in JavaScript. For code running using Node.js APIs, converting between base64-encoded strings and binary data should be performed using `Buffer.from(str, 'base64')` and`buf.toString('base64')`.

But probably we should reasonably ignore this?